README.md

```diff
@@ -187,7 +187,13 @@ refund = mollie_client.refunds.on(payment).create({
 
 ## Oauth2 ##
 
-Run the oauth2 examples.
+Oauth authentication process redirects back to your application.
+Therefore you should expose your local web server (the examples) as public urls.
+
+A webservice like [ngrok.com](https://ngrok.com/) can help you with that. Make sure to set REDIRECT_URI accordingly.
```

Comment:
Still missing pointers on client ID and client secret. Just add a pointer to https://docs.mollie.com/oauth/getting-started I think?